outscale/data_source_outscale_access_key_test.go

```diff
@@ -50,21 +28,11 @@ func TestAccOutscaleDataSourceAccessKey_withFilters(t *testing.T) {
 	})
 }
 
-func testAccClientAccessKeyDataSourceBasic() string {
-	return `
-		resource "outscale_access_key" "outscale_access_key" {}
-
-		data "outscale_access_key" "outscale_access_key" {
-			access_key_id = outscale_access_key.outscale_access_key.id
-		}
-	`
-}
-
 func testAccClientAccessKeyDataSourceWithFilters() string {
 	return `
 		resource "outscale_access_key" "outscale_access_key" {}
 
-		data "outscale_access_key" "outscale_access_key" {
```

Comment:
But you change outscale_access_key to access_key, which change the resource name, so it's a breaking change.

Reply:
No, i change the name of the example, not the datasource. 

But this PR is just for test somme thing actually ^^"